cmds/record.c

```diff
@@ -136,7 +136,7 @@ char * get_libmcount_path(struct opts *opts)
 	if (access(lib, F_OK) == 0)
 		return lib;
 #endif
-	strcpy(lib, libmcount);
+	strncpy(lib, libmount, sizeof(lib));
```

Comment:
Thanks, but `sizeof(lib)` seems to be always 8 because `lib` is a pointer type, not an array.

Reply:
Thanks I'll change the argument right away

Reply:
`libmcount` variable receives hard-coded string.

In my opinion, it is not vulnerable.

https://github.com/namhyung/uftrace/blob/f35749873f05e2e8d7fc4499ca45c07ea62ec536/cmds/record.c#L100-L116

Reply:
I don't think this issue would cause severe security threat either. 
But I thought specifying the maximum byte using `strncpy` 
would be the best practice we can make :)

Reply:
Other parts of the code in uftrace (for example, `utils/fstack.c`) 
used `strncpy` to copy a fixed length string, so I thought
this could be ok as well.

https://github.com/namhyung/uftrace/blob/master/utils/fstack.c#L1947